components/skus/browser/skus_sdk_service.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_SKUS_BROWSER_SKUS_SDK_SERVICE_H_
+#define BRAVE_COMPONENTS_SKUS_BROWSER_SKUS_SDK_SERVICE_H_
+
+#include <memory>
+#include <string>
+
+#include "base/memory/weak_ptr.h"
+#include "brave/components/skus/browser/brave-rewards-cxx/src/wrapper.h"
+#include "brave/components/skus/common/skus_sdk.mojom.h"
+#include "components/keyed_service/core/keyed_service.h"
+
+class PrefService;
+
+namespace network {
+class SharedURLLoaderFactory;
+}  // namespace network
+
+// This service is wrapping the calls to our SKU SDK
```

Comment:
I think we need better descriptions for these classes and the link isn't helpful because there is no description there either

Reply:
Updated descriptions! See https://github.com/brave/brave-core/pull/10847/commits/85aaf7d6bd1a19a5eda663ac17f5db0632aca103 for more info